### DISCUSSION TITLE
Does updateAsDraft also handle relationships

### DISCUSSION THREAD

**Discussion by T3mmyKay** [oddvalue/laravel-drafts#58] [Q&A] [answered]
I wanted to ask if the updateAsDraft handles saving relationship because for example I am using filamentphp and I am trying to updateAsDraft and also save relationship data to the new draft

**Comment by oddvalue** [comment, marked as answer]
At the moment, no. Relations in the `$draftableRelations` array will be moved from the published revision to the draft when that draft is published but the relations won't be cloned with every draft.
